worldedit-core/src/main/java/com/sk89q/worldedit/command/ApplyBrushCommands.java

```diff
@@ -111,11 +116,19 @@ public void forest(CommandParameters parameters,
         name = "item",
         desc = "Use an item"
     )
+    @CommandPermissions("worldedit.brush.item")
     public void item(CommandParameters parameters,
                      Player player, LocalSession localSession,
                      @Arg(desc = "The type of item to use")
-                         BaseItem item) throws WorldEditException {
-        setApplyBrush(parameters, player, localSession, new ItemUseFactory(item));
+                         BaseItem item,
+                     @Arg(desc = "The direction to face applying the item", def = "up")
+                     @Direction(includeDiagonals = true)
+                         BlockVector3 direction) throws WorldEditException {
+        player.print(TextComponent.of("Warning: ", TextColor.RED)
+                .append(TextComponent.of("This brush simulates item usages. Its effects may not work on all platforms, " +
+                        "may not be undoable, and may cause strange interactions with other mods/plugins. Use at your own risk.")));
+        setApplyBrush(parameters, player, localSession, new ItemUseFactory(item,
+                com.sk89q.worldedit.util.Direction.findClosest(direction.toVector3(), com.sk89q.worldedit.util.Direction.Flag.ALL)));
```

Comment:
Probably should just add a new converter in `DirectionConverter` that can handle `@Direction Direction`. Though that may not look great as a parameter. Maybe we should rename the `@Direction` annotation?